torch/_tensor_docs.py

```diff
@@ -1068,6 +1068,13 @@ def scale_channels(input, scale):
 See :func:`torch.floor`
 """)
 
+add_docstr_all('floor_divide',
+               r"""
+floor() -> Tensor
```

Comment:
This should be `floor_div(other) -> Tensor`; that might be causing the test failures. But it sounds like we're removing the method variant to be consistent with numpy so this may not apply.